app/controllers/hud_reports/base_controller.rb

```diff
@@ -79,6 +81,58 @@ def download
       end
     end
 
+    # Returns a frozen hash of available report versions for HUD reports.
+    # Each version entry contains:
+    #   - slug: The version identifier (e.g. :fy2024)
+    #   - active: Boolean indicating if this version is currently active
+    #
+    # The active version is determined by default_report_version:
+    # - FY 2024 is active when default_report_version is :fy2024
+    # - FY 2026 is active when default_report_version is :fy2026
+    #
+    # Historical versions (FY 2020, 2022, 2023) are always inactive.
+    # @return [Hash] A frozen hash mapping version names to their configuration
+    # @example
+    #   {
+    #     'FY 2020' => { slug: :fy2020, active: false },
+    #     'FY 2024 (current)' => { slug: :fy2024, active: true }
+    #   }
+    def available_report_versions
+      @available_report_versions ||= {}.tap do |version|
+        version['FY 2020'] = { slug: :fy2020, active: false }
+        version['FY 2022'] = { slug: :fy2021, active: false }
+        version['FY 2023'] = { slug: :fy2023, active: false }
+        # Keep FY 2024 on production until 10/1/2025
+        # Keep FY 2024 on staging until 9/1/2025
+        case default_report_version
+        when :fy2024
+          version['FY 2024 (current)'] = { slug: :fy2024, active: true }
+          version['FY 2026'] = { slug: :fy2026, active: false }
+        when :fy2026
+          version['FY 2024'] = { slug: :fy2024, active: false }
+          version['FY 2026 (current)'] = { slug: :fy2026, active: true }
+        end
+      end.freeze
+    end
+    helper_method :available_report_versions
+
+    # Determines the default report version based on environment and date.
+    # - In production: Uses FY 2024 until October 1, 2025, then switches to FY 2026
+    # - In staging: Uses FY 2024 until September 1, 2025, then switches to FY 2026
+    # - In all other environments: Uses FY 2026
+    # @return [Symbol] The default report version (:fy2024 or :fy2026)
+    # Override as necessary in subclasses
+    def default_report_version
+      return :fy2024 if Rails.env.production? && Date.current <= '2025-10-01'.to_date
+      return :fy2024 if Rails.env.staging? && Date.current <= '2025-09-01'.to_date
+
+      :fy2026
+    end
```

Comment:
Suggestion for better code clarity:
```suggestion
    def default_report_version
      comparison = Rails.env.production? ? '2025-10-01' : '2025-09-01'
      return :fy2024 if Date.current <= comparison_date.to_date

      :fy2026
    end
````

Reply:
@ttoomey that logic isn't the same.  It is important that we check for `Rails.env.staging?` to allow test and development to function immediately.

Leaving as is.